### PR TITLE
Add new book

### PR DESCRIPTION
**Pull request by NenadJeckovic**
Added new book form to admin books page with success and failure response messages. Tests included in AdminBooksPage tests.

### CODE REVIEW

File: src/pages/AdminBooksPage/AdminBooksPage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      <MemoryRouter>
+        <AdminBooksPage />
+      </MemoryRouter>
+    );
+
+    const newBookButton = await screen.findByTestId("new-book-button");
+
+    expect(newBookButton).toBeInTheDocument();
+  });
+});
```
Comment (ssukovic) [inline src/pages/AdminBooksPage/AdminBooksPage.test.tsx:18]: You should add one test that shows what happens when button is clicked and goes through the whole flow, fills the form, submits the book and checks if success notification shows up. 
NewBookForm.test covers only the form, we should also test the integration and the whole flow.

**Review by ssukovic — CHANGES_REQUESTED** [review]
Two additional points:

- Some error handling should be added while we're moving these functionalities, currently when api for adding new books fails nothing happens. In the file NewBookForm similar to onSuccessCallback create onFailureCallback which doesn't hide the form and displays a failure message to the user "Error when adding a new book. Please try again later". 
- It would be helpful and a good practice if you added descriptions to your PRs, just simple outline explaining what you did and how to test it, so reviewers can see at a glance what's going on without having to look for the related ticket.

**Review by ssukovic — APPROVED** [review]
Great job 🎉

File: src/components/Notification/index.tsx [hunk 2]
```diff
@@ -28,7 +29,7 @@ const Notification = () => {
           warning={warning}
         />
       ) : (
-        <SuccessScreen />
+        <>{warning ? <WarningScreen /> : <SuccessScreen />}</>
```
Comment (ssukovic) [inline src/components/Notification/index.tsx:32]: Good catch, I forgot about the mobile screen 😄

**Review by dusanpetrovicpd — APPROVED** [review]

**Pull request merged**